flink-docs/src/test/java/org/apache/flink/docs/rest/OpenApiSpecGeneratorTest.java

```diff
@@ -131,4 +129,28 @@ public List<Tuple2<RestHandlerSpecification, ChannelInboundHandler>> initializeH
                     Tuple2.of(new TestEmptyMessageHeaders("operation1"), null));
         }
     }
+
+    @Test
+    void testAdditionalFields(@TempDir Path tmpDir) throws Exception {
+        final Path file = tmpDir.resolve("openapi_spec.yaml");
+        OpenApiSpecGenerator.createDocumentationFile(
+                "title", new TestAdditionalFieldsRestEndpoint(), RuntimeRestAPIVersion.V0, file);
+        final String actual = new String(Files.readAllBytes(file), StandardCharsets.UTF_8);
+        assertThat(actual)
+                .contains(
+                        "    AdditionalFieldsRequestBody:\n"
+                                + "      type: object\n"
+                                + "      additionalProperties:\n"
+                                + "        type: string");
```

Comment:
nit: Do we actually want to check for spaces? Isn't it easy to break?

Reply:
I've adjusted the test to work directly on the java representation